src/Main.js

```diff
@@ -18,15 +24,35 @@ class Main extends React.Component {
         keyword={beast.keyword}
         title={beast.title}
         handleRenderModal={this.props.handleRenderModal}
+        handleSelectForm={this.props.handleSelectForm}
+        hornsSelectedState={this.props.hornsSelectedState}
       />
 
     ));
+    let selectedBeastArr = beastArr;
+    if (this.state.hornsSelectedState === 'all') {
+      selectedBeastArr = beastArr
+    }
+
+    if (this.state.hornsSelectedState === '1') {
+      selectedBeastArr = (beastArr.filter(beast => (beast.state.hornCount === '1') === true))
+    }
+
+    if (this.state.hornsSelectedState === '2') {
+      selectedBeastArr = (beastArr.filter(beast => (beast.state.hornCount === '2') === true))
+    }
 
+    if (this.state.hornsSelectedState === '3') {
+      selectedBeastArr = (beastArr.filter(beast => (beast.state.hornCount === '3') === true))
+    }
 
+    if (this.state.hornsSelectedState === 'many') {
+      selectedBeastArr = (beastArr.filter(beast => (parseInt(beast.state.hornCount) >= 4) === true))
+    }
 
```

Comment:
- We will want to simplify this and extract this logic out of render if possible 👍 
- Grab a TA to see if they can help
- We can create a method outside of render that will handle our filter() logic. 
- We can do this using 1 filter() as well which will be best